access-control-app/src/cmr/access_control/api/routes.clj

```diff
@@ -109,350 +109,351 @@
 (defn- search-for-groups
   [ctx headers params]
   (mt/extract-header-mime-type #{mt/json mt/any} headers "accept" true)
-  (-> (group-service/search-for-groups ctx (dissoc params :token))
-      common-routes/search-response))
+  (common-routes/search-response
+   (group-service/search-for-groups ctx (dissoc params :token))))
+
 
 ;;; ACL Route Functions
 
-(defn- create-acl
-  "Returns a Ring response with the result of trying to create the ACL with the given request body."
-  [request-ctx headers body]
-  (validate-content-type headers)
-  (acl-schema/validate-acl-json body)
-  (try
+  (defn- create-acl
+    "Returns a Ring response with the result of trying to create the ACL with the given request body."
+    [request-ctx headers body]
+    (validate-content-type headers)
+    (acl-schema/validate-acl-json body)
+    (try
+      (->> (json/parse-string body)
+           util/map-keys->kebab-case
+           (acl-service/create-acl request-ctx)
+           util/map-keys->snake_case
+           api-response)
+      (catch com.fasterxml.jackson.core.JsonParseException e
+        (errors/throw-service-error :bad-request (str "Json parsing error: " (.getMessage e))))))
+
+  (defn- update-acl
```

Comment:
spacing off here too.

Reply:
is this spacing done by calva or did something else set the spaces?